RoboJackets/igvc-software
# Feat/multiclass segmentation

# Implements the Multiclass Segmentation model 

This PR does the following:
- Implements the Multiclass Segmentation model 

Fixes #747 

# Testing steps (with bag file)
1. Make sure to have the model downloaded and placed in `igvc_sandbox/models/MC_model.pth` (You can either [train your own model](https://github.com/RoboJackets/igvc-software/tree/master/igvc_perception/src/multiclass_segmentation) or download the one located in  `Shared Drives > RoboNav > Previous Year Resources > 2019-2020 Season > Software > Collab Notebooks > MC_Model.pth`)
2. Have lines 89-92 in `multiclass_segmentation.py` commented out
3. run `roscore`
4. Run the Bag file with something like: `rosbag play ~/path/to/file/jessii_JOHNSCREEK_2020-02-22-17-39-35_3.bag -l`
5. Run the multiclass with: `roslaunch igvc_perception multiclass_segmentation.launch`
6. Launch RVIZ and add the `/semantic_segmentation/usb_cam_(center/right/left)/...` topic

Expectation: 
We see the segmented display

# Self Checklist
